lib/simulation/openfisca/mapping/index.js

```diff
@@ -162,8 +162,18 @@ var transform = function(src, schema, situation) {
     return result;
 };
 
+var isIndividuValid = exports.isIndividuValid = function(individu, situation) {
```

Comment:
This helper doesn't feel at home in `index.js`. Put it in a separate file.

The best would have been `individu`, but it's already used as a map. What about `common`, maybe?